src/net/sourceforge/kolmafia/KoLAdventure.java

```diff
@@ -1504,9 +1500,16 @@ public boolean canAdventure() {
       String today = HolidayDatabase.getHoliday();
       return switch (this.adventureNumber) {
         case AdventurePool.DRUNKEN_STUPOR -> KoLCharacter.isFallingDown();
-        case AdventurePool.SSPD_STUPOR -> today.contains(holiday)
-            && KoLCharacter.getInebriety() >= 26;
-        default -> (holiday == null) ? true : today.contains(holiday);
+        case AdventurePool.SSPD_STUPOR -> {
+          if (today.contains(holiday)
+              || today.equals("Drunksgiving") && KoLCharacter.getInebriety() >= 26) {
+            yield true;
```

Comment:
Is this correct? && has a higher operator precedence than ||.
I think you mean
```(today.contains(holiday) || today.equals("Drunksgiving) && KoLCharacter.getInebriety() >= 16```

Or maybe even ```today.contains("Drunksgiving")``` although when is that going to next happen? :)

Reply:
Good spot!